jsonschema/_format.py

```diff
@@ -412,18 +444,43 @@ def is_relative_json_pointer(instance):
 
 
 try:
-    import uritemplate.exceptions
+    import uri_template
```

Comment:
Can you pull this out into its own PR (which likely can merge immediately).

Reply:
Moved to https://github.com/Julian/jsonschema/pull/819